packages/cozy-harvest-lib/src/components/DumbTriggerManager.jsx

```diff
@@ -99,7 +99,7 @@ export class DumbTriggerManager extends Component {
 
   async handleSubmit(data = {}) {
     const { client, flow, konnector, vaultClient, t } = this.props
-    const { account } = this.state
+    const account = this.state?.account || flow.account
```

Comment:
`state` est toujours def non ? 🤔 

Est-ce que ne ferait pas mieux de laisser `const { account } = this.state` et de faire un `const account = props.account || props.flow.account` à la place de `const { account } = props` dans le constructor ?

Mais du coup ça pose question, pourquoi avoir un account et un flow.account dans les props ? 🤔 D'ailleurs ça parait legitime d'après le descriptif des propTypes d'avoir un account undefined, c'est donc étonnant d'avoir un account undefined et un flow.account defined, non ? 🤔

Reply:
> state est toujours defined non ?

Oui, c'est corrigé

> c'est donc étonnant d'avoir un account undefined et un flow.account defined, non ? 

On a le cas `account undefined` et `flow.account` defined justement quand l'utilisateur a fait une première tentative (-> donc account et trigger créé) et qu'il refait une nouvelle tentative en restant sur le même écran de login

> Est-ce que ne ferait pas mieux de laisser ...

Oui c'est probablement mieux et moins enfoui au milieux du code comme ça, corrigé

Reply:
@doubleface si tu peux rajouter un petit commentaire quelque part ou de la doc pour expliquer dans le code comment on peut se retrouver dans le cas `account undefined` et `flow.account defined` c'est top 👍